interface/therapy_groups/therapy_groups_views/appointmentComponent.php

```diff
@@ -0,0 +1,50 @@
+<div class="row" xmlns="http://www.w3.org/1999/html">
```

Comment:
Needs a standard gpl header with your copyright and author tag. See here for example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock